less/list.less

```diff
@@ -98,35 +89,28 @@
     color: @item-color-inverted;
     border-radius: 50%;
   }
-  // --------------------------------------------------
 
-  // --------------------------------------------------
   // Animated list
   // --------------------------------------------------
-  &.is-animated-list {
-    .list__widget {
-      overflow: hidden;
-    }
+  &.is-animated-list &__widget {
+    overflow: hidden;
+  }
 
-    .list__container .list__item {
-      position: relative;
-      right: -100%;
-      opacity: 0;
-      .transition(all .8s ease-in-out);
-    }
+  &.is-animated-list &__item {
+    position: relative;
+    right: -100%;
+    opacity: 0;
+    .transition(all .8s ease-in-out);
+  }
 
-    .list__container .list__item.is-animating {
-      right: 0;
-      opacity: 1;
-    }
+  &.is-animated-list &__item.is-animating {
+    right: 0;
+    opacity: 1;
```

Comment:
Please move this block to line 104 with a selector of `&.is-animating`.

Reply:
Done